internal/descriptor/descriptor.go

```diff
@@ -37,3 +37,11 @@ func ShortDigest(desc ocispec.Descriptor) (digestString string) {
 	}
 	return digestString
 }
+
+func GetTileOrMediaType(desc ocispec.Descriptor) (name string, isTitle bool) {
```

Comment:
Also requires since it's public function

```suggestion
func GetTitleOrMediaType(desc ocispec.Descriptor) (name string, isTitle bool) {
```

Reply:
Done